lib/features/server/config-server.js

```diff
@@ -18,6 +18,7 @@ const { AudioPlugin } = require('../../audio/server/plugin');
 const { RoomsPlugin } = require('../../rooms/server/plugin');
 const { AdminPlugin } = require('../../admin/server/plugin');
 const { TeamsPlugin } = require('../../teams/server/plugin');
+const { RewardsPlugin } = require("../../rewards/server/plugin");
```

Comment:
single quotes

Reply:
still seeing the double quotes:
`require("../../rewards/server/plugin");` > `require('../../rewards/server/plugin');`